validator/validate_api.py

```diff
@@ -293,22 +299,27 @@ def validation_api_entry():
 
     response = args.get("response", None)
     uid = args.get("uid", None)
+    feature_weights_set_id= args.get("feature_weights_set_id", 'd3732be6-a759-43aa-9e1a-3e9bd94f8b6b')
+
     parser_params = {
         key: make_tristate(args.get(key, val), val)
         for key, val in PARSER_DEFAULTS.items()
     }
+
     feature_weight_dict = OrderedDict(
         {
             key: make_tristate(args.get(key, val), val)
-            for key, val in VALIDITY_FEATURE_DICT.items()
+            for key, val in current_app.df["feature_weights"][feature_weights_set_id].items()
         }
     )
 
     start_time = time.time()
     return_dictionary = validate_response(
-        response, uid, feature_weight_dict, **parser_params
+        response, uid, **parser_params
     )
```

Comment:
This feature_weight_dict is never used - and no id exists for it, to pass to validate_response(). Remove the loop over args, and just pass the feature_weights_Set_id to validate_response()